entx/gql_hooks.go

```diff
@@ -88,6 +90,36 @@ var (
 		}
 		return nil
 	}
+
+	addNodesToConnections = func(_ *gen.Graph, s *ast.Schema) error {
+		for _, t := range s.Types {
+			if !strings.HasSuffix(t.Name, "Connection") {
+				continue
+			}
+
+			existingFields := []string{"edges", "pageInfo", "totalCount"}
+			missingField := false
+			for _, f := range t.Fields {
+				if !slices.Contains(existingFields, f.Name) {
+					missingField = true
```

Comment:
It's not really a huge improvement, but we could break after setting this.

Reply:
I thought about it, but given there are only 3 fields on a Connection I wasn't to worried about it.